crypto/rand/rand_unix.c

```diff
@@ -93,6 +93,27 @@ static uint64_t get_timer_bits(void);
 # error "UEFI and VXWorks only support seeding NONE"
 #endif
 
+#if defined(OPENSSL_SYS_VXWORKS)
```

Comment:
This seems really bad.  Returning "it's all okay" when, in fact, there is no entropy available.  Is there a better way to do this?

Reply:
good question, we plan to fix this soon but for the moment there's not much we can do i assume? woudl it be ok to leave it in temporary until we come to a conclusion on how to handle entropy on vxworks (7)?

Reply:
@richsalz are you convinced by our conversation at https://github.com/openssl/openssl/pull/7569#issuecomment-449049572 and following?

Reply:
as long as it will not fail but silently let rand_bytes act like it works, sure.